### DISCUSSION TITLE
Why do we need schwas before an L or R? Aren’t they implied?

### DISCUSSION THREAD

**Discussion by trosel** [Shavian-info/readlex#38] [Spelling principles]
Take “dial”. Why is it 𐑛𐑲𐑩𐑤 instead of just 𐑛𐑲𐑤? 

Is it even possible to pronounce that L without a little schwa before it? If not, why even write the schwa? It seems included with the L inherently, no?

I understand there are non-rhotic accents, but the same question goes for Rs. A schwa before the R is basically included, no?

**Comment by Shavian-info** [comment]
I've converted this to a discussion since it's not about an error or problem in the Read Lexicon.

The use of 𐑩 before 𐑤 and 𐑮 follows RP pronunciation, as reflected in dictionaries like Cambridge Pronouncing Dictionary and the Oxford English Dictionary. Refer to [Spelling Principle 1](https://readlex.pythonanywhere.com/spellingprinciples/). The difference is subtle in some cases, but is there.

**Reply by trosel** [reply]
Thank you, I always forget that discussions exist here now!

It is indeed tricky because other places where there is definitely a schwa “whale”, “kale”, etc do not show a schwa in the IPA. The L seems to include a schwa by default. That is what I am speaking of here

**Comment by trosel** [comment]
I believe that this video explains at 3:01 https://youtu.be/gtnlGH055TA?t=182

It has to do with "pre-L breaking" vowels.

**Reply by Shavian-info** [reply]
Yep, and the ReadLex follows the RP approach as recorded in relevant dictionaries. This distinguishes real/reel, phial/file, viol/vile, for example, even though in my own accent they're basically merged.
